packages/common/src/hooks/useTrackMetadata.ts

```diff
@@ -59,6 +63,17 @@ export const useTrackMetadata = ({
       id: TrackMetadataType.MOOD,
       label: 'Mood',
       value: mood
+    },
+    {
+      id: TrackMetadataType.BPM,
+      label: 'BPM',
+      value: bpm
+    },
+    {
+      id: TrackMetadataType.KEY,
+      label: 'Key',
+      // TODO: KJ - Might need a map for this to map to key options
```

Comment:
do we want to solve this now?

Reply:
was going to wait to see the format that the keys were being stored as to make the map

Reply:
should see it on staging now right?

Reply:
the `musical_key` key is there, but I don't have examples of the values yet